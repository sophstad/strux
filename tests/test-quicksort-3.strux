int main() {
	num[] a = [1.2, 4.4, 3.5, 6.5, 7.5, 2.3, 99.5, 23.9, 37.9];
  	a.fquickSort();
	int i = 0;
	for (i = 0; i < 9; i = i + 1) {
		print(a[i]);
	}
	return 0;
}
