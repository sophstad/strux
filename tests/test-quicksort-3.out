1.200000
2.300000
3.500000
4.400000
6.500000
7.500000
23.900000
37.900000
99.500000
